src/components/Notice/Notice.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Text from '../Text';
+import { borderRadius } from '../style/borders';
+import { fontSize } from '../style/fonts';
+import Warning from '../Icon/Icons/Warning';
+import Cross from '../Icon/Icons/Cross';
+
+const NoticeWrapper = styled.div`
+  border: 1px solid #a59638;
+  color: #625920;
+  background: #fdf8d8;
+  border-radius: ${borderRadius};
+  font-size: ${fontSize};
+  padding: 16px 16px;
+  display: flex;
+  position: relative;
+`;
+
+const WarningIcon = styled(Warning)`
```

Comment:
Love what you are doing here to avoid adding extra wrappers just for paddings on margins ❤️

Reply:
hehehe got this from Dinos! I also liked this way 👏